prime-router/src/main/resources/db/migration/V22__add_look_tables.sql

```diff
@@ -20,6 +20,7 @@
   lookup_table_version_id BIGSERIAL PRIMARY KEY,
   table_name character varying(512) NOT NULL,
   table_version INTEGER NOT NULL CHECK (table_version >= 0),
+  table_Sha256 character varying(512) NULL,
```

Comment:
This should be unique. or unique with `table_name`

Reply:
Corrected the name to table_sha256 and move to V30__add_lookup_table.sql.  Thank you.